src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -387,7 +389,8 @@ public boolean parseArguments(final PrintStream messageStream, final String[] ar
 
         //check if special short circuiting arguments are set
         if (isSpecialFlagSet(parsedArguments, SpecialArgumentsCollection.HELP_FULLNAME)) {
-            messageStream.print(usage(true));
+            // TODO: should a new SpecialArgumentCollection being set to display hidden arguments?
```

Comment:
I added the "showHidden" special argument, and it is annotated with Advanced. I'm up for handling `@Advance` in the command line (actually it was in my plans), but maybe it will be better to do it in a different PR to get this into.